tests/scripts/initialize_database.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+docker ps -a
+docker-compose up
```

Comment:
you need to add -d or you will be blocked at this step

Reply:
to the docker ps -a ? or to the docker-compose up ?

Reply:
docker-compose up